conda_libmamba_solver/index.py

```diff
@@ -409,3 +411,26 @@ def _process_query_result(
 
 # for conda-build
 _CachedLibMambaIndexHelper = lru_cache(maxsize=None)(LibMambaIndexHelper)
+
+
+def _LibMambaIndexForCondaBuild(*args, **kwargs):
```

Comment:
The type change from class to function for the index helper is a little miffy, I'd rather see use a subclass and then override `context.offline` in the `_CachedLibMambaIndexHelper.__init__` when calling `super`.

Reply:
Ok, I like it.